src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Status } from '../../enum/Status';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  filterBy: Status,
+  setFilterBy: (filterBy: Status) => void,
+};
+
+export const Footer: React.FC<Props> = ({ setFilterBy, filterBy, todos }) => {
+  const completedTodo = todos.filter(todo => todo.completed);
+  const activeTodosQuantity = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <>
+      {todos.length > 0 && (
+        <footer className="todoapp__footer">
+          <span className="todo-count">
+            {`${activeTodosQuantity} items left`}
+          </span>
+
+          <nav className="filter">
+            <a
+              href="#/"
+              className={classNames('filter__link', {
+                selected: filterBy === Status.all,
+              })}
+              onClick={() => setFilterBy(Status.all)}
```

Comment:
it's better to move this callback to an own handler function

Reply:
ok.
but still it will be callback, smth like onClick= {() => handleFilterStatus(Status.all)}

or you can give me some advice?